hash_practice/exercises.py

```diff
@@ -1,19 +1,73 @@
 
+from audioop import reverse
+from copy import copy
+from curses import keyname
+from tabnanny import check
+
+
 def grouped_anagrams(strings):

```

Comment:
✨ Nice work. We can say this is O(nm) where n is the length of `strings` and `m` is the length of the longest word in `strings` for time complexity. We could also reduce this down to O(n) assuming all strings are valid English words and therefore of relatively negligible length